### ISSUE TITLE
Feature request - multiple warden/token/allowed per request

### ISSUE THREAD

**Issue by waynerobinson** [ory-am/hydra#264]
Request round-trip times can be annoying and not all permission checks are for single resources.

It would be nice to be able to include multiple token/allowed requests in a single call.

**Comment by arekkas** [comment]
Good idea regarding policies, but do you really need to checking for multiple tokens per request?

**Comment by waynerobinson** [comment]
Nope. One token is fine, as is a simplified response that basically
includes an OK/Fail for each check.

**Comment by arekkas** [comment]
ok, what would the default behaviour be, allow access if one of the resources is allowed? what if one policy explicitly is set to deny?

**Comment by waynerobinson** [comment]
One result per resource/action check in an array.

**Comment by arekkas** [comment]
That makes sense!

**Comment by arekkas** [comment]
@waynerobinson is this something you want for the stable release, or is it more like a "nice to have"?

**Comment by waynerobinson** [comment]
No, this is just a nice to have for now. We can make do without. We're still working through the best way to get Warden to work in our app and we can always build our client interface to be easily upgradable to be able to request multiple policies later.

**Comment by arekkas** [comment]
Thanks for the quick response. I'm sorting out priorities for the stable release of Hydra and wanted to check with you if it's something you want included there. From what you said I take it as a low prio :)

**Comment by subandroid** [comment]
it looks like we may have a need for this and i'm looking into how best to implement this. let me know if you guys have any concern/suggestions.

**Comment by joshuarubin** [comment]
This would be hugely beneficial for us as well. Right now we have to loop through several possible actions and check with warden for each of them for a single request to our api.
